R/pkg/R/mllib.R

```diff
@@ -197,7 +201,7 @@ print.summary.GeneralizedLinearRegressionModel <- function(x, ...) {
   invisible(x)
   }
 
-#' Make predictions from a generalized linear model
+#' predict
```

Comment:
@jkbradley The reason I want add title here is the title of current documentation is like  `Make predictions from a generalized linear model` not `predict`. 
![qq20160613-0 2x](https://cloud.githubusercontent.com/assets/3419881/16033524/4ab5e288-31c1-11e6-892c-c9c15258cc05.png)
But with adding title `predict`, it looks like this
![qq20160613-1 2x](https://cloud.githubusercontent.com/assets/3419881/16033671/24ab8fce-31c2-11e6-86a4-7b7771c10451.png)
So which one do you think is better?